electric_emission_cost/costs.py

```diff
@@ -365,12 +377,26 @@ def get_charge_df(
         for key, value in charge_dict.items()
         if any(k in key for k in ["electric_customer", "gas_customer"])
     }
+    if scale_charges:
+        # scale the fixed charges by the number of timesteps in the month
+        month = start_dt.month
+        year = end_dt.year
+        mins_in_month = (
+            (dt.date(year, month + 1, 1) - dt.date(year, month, 1)).days * 24 * 60
+        )
+        bins_in_month = mins_in_month / res_binsize_minutes
+        scale_factor = ntsteps / bins_in_month
+
+        for key, value in charge_dict.items():
+            if any(k in key.split("_") for k in ["customer", "demand"]):
+                charge_dict[key] = value * scale_factor
 
     if keep_fixed_charges:
         # replace the fixed charge in charge_dict with its time-averaged value
         for key, value in fixed_charge_dict.items():
-            charge_dict[key] = np.ones(ntsteps) * value / ntsteps
-
+            arr = np.zeros(ntsteps)
```

Comment:
Unless I am misunderstanding the logic, won't `keep_fixed_charges` overwrite the changes made in `scale_charges`? Because it's adding the value to just the first element of the array.